frontend/src/components/shared/FullScreenWrapper/FullScreenWrapper.scss

```diff
@@ -0,0 +1,32 @@
+.streamlit-container .stFullScreenFrame--expanded {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  bottom: 0px;
+  right: 0px;
+  overflow: auto;
+  background: white;
+  z-index: 1000;
+  padding: 30px;
+  padding-top: 80px;
```

Comment:
Use the same value as the header height here. I believe that's 3.5rem (which includes some margin).

Also, please put 3.5rem as a variable in variables.scss and use it in header.scss and here.